metrics/core/src/main/java/org/hyperledger/besu/metrics/BesuMetricCategory.java

```diff
@@ -40,9 +42,11 @@
   public static final Set<MetricCategory> DEFAULT_METRIC_CATEGORIES;
 
   static {
-    // Why not ROCKSDB and KVSTORE_ROCKSDB_STATS?  They hurt performance under load.
+    // Why not ROCKSDB and KVSTORE_ROCKSDB_STATS, KVSTORE_PRIVATE_ROCKSDB_STATS?  They hurt
```

Comment:
Comment might be wrong, where is the `ROCKSDB` in the enum?

Reply:
done.